build.gradle

```diff
@@ -245,10 +245,11 @@ repositories {
 dependencies {
     minecraft "net.minecraftforge:forge:${forge_version}"
 
-   	//compileOnly fg.deobf("mezz.jei:jei-${mc_version}:${jei_version}:api")
-   	//runtimeOnly fg.deobf("mezz.jei:jei-${mc_version}:${jei_version}")
+//   	compileOnly fg.deobf("mezz.jei:jei-${mc_version}:${jei_version}:api")

```

Comment:
This line and the one under it can still be used the naming convention for the files changed: https://github.com/CodersDownUnder/FlowerSeeds/blob/69d6b490aad7b33b3d6aa892685556143f3a5ebb/build.gradle#L124-L125